.vscode/settings.json

```diff
@@ -20,7 +20,7 @@
 		"source.organizeImports.biome": "explicit"
 	},
 	"editor.formatOnPaste": true,
-	"editor.formatOnSave": true,
+	"editor.formatOnSave": false,
```

Comment:
This was causing issues ... sort of a "double save" ... so I disabled it and now VSC is happy